components/engine_rocks/Cargo.toml

```diff
@@ -37,7 +37,8 @@ serde_derive = "1.0"
 coarsetime = "0.1"
 
 [dependencies.rocksdb]
-git = "https://github.com/tikv/rust-rocksdb.git"
+git = "https://github.com/tabokie/rust-rocksdb.git"
```

Comment:
Can you send a rust-rocksdb PR for the change? btw if you use cargo patch, you only need to change the top level Cargo.toml. https://doc.rust-lang.org/edition-guide/rust-2018/cargo-and-crates-io/replacing-dependencies-with-patch.html

Reply:
tikv/rust-rocksdb#526 PTAL